src/Akka.Persistence.Redis.Tests/RedisJournalPerfSpec.cs

```diff
@@ -39,7 +39,7 @@ class = ""Akka.Persistence.Redis.Journal.RedisJournal, Akka.Persistence.Redis""
         public RedisJournalPerfSpec(ITestOutputHelper output, RedisFixture fixture) : base(Config(fixture, Database), nameof(RedisJournalPerfSpec), output)
         {
             EventsCount = 1000;
-            ExpectDuration = TimeSpan.FromMinutes(10);
+            ExpectDuration = TimeSpan.FromMinutes(2);
```

Comment:
Does it really need 2 minutes for a test to expect to finish? Have you tested this locally?

Reply:
It actually need 10 minutes. Locally, 2 minutes was enough but with the test server 10 minutes seems to be what it needs to complete!